articles/index.yml

```diff
@@ -22,107 +22,112 @@ landingContent:
+            url: quickstarts/search.md
 
-  # Samples
-  - title: Q# samples
+
+  # Reference card
+  - title: Quantum computing concepts
     linkLists:
-      - linkListType: how-to-guide
+      - linkListType: concept
         links:
-          - text: Three simple algorithms
-            url: https://docs.microsoft.com/samples/microsoft/quantum/simple-quantum-algorithms
-          - text: Searching with Grover's algorithm
-            url: https://docs.microsoft.com/samples/microsoft/quantum/searching-with-grovers-algorithm
-          - text: Quantum Teleportation
-            url: https://docs.microsoft.com/samples/microsoft/quantum/quantum-teleportation/
-          - text: All Samples
-            url: https://docs.microsoft.com/samples/browse/?languages=qsharp
+          - text: Quantum computing concepts
+            url: concepts/index.md
 
  
   # Development
-  - title: Q# language & development techniques
+  - title: Q# programming language
     linkLists:
       - linkListType: learn
         links:
-        - text: Q# language introduction
-          url: language/index.md
-      - linkListType: how-to-guide
+          - text: Q# language introduction
+            url: language/index.md
+          - text: Q# development techniques
+            url: techniques/index.md
+
+
+  # Reference card
+  - title: Target machines
+    linkLists:
+      - linkListType: overview
         links:
           - text: Quantum simulators & host applications
             url: machines/index.md
-      - linkListType: how-to-guide
+      - linkListType: architecture
         links:
-          - text: Q# program overview
-            url: techniques/file-structure.md
-          - text: Operations & functions
-            url: techniques/operations-and-functions.md
-          - text: Local variables
-            url: techniques/local-variables.md
-          - text: Working with qubits
-            url: techniques/working-with-qubits.md
-          - text: 'Put it together: Teleportation'
-            url: techniques/putting-it-all-together.md
-
+          - text: Full-state simulator
+            url: machines/full-state-simulator.md
+          - text: Resources estimator
+            url: machines/resources-estimator.md
+          - text: Trace simulator
+            url: machines/qc-trace-simulator/index.md
+          - text: Toffoli simulator
+            url: machines/toffoli-simulator.md
   
-  # Reference card
-  - title: Quantum computing concepts
+  # Samples
+  - title: Q# learning resources
     linkLists:
-      - linkListType: concept
+      - linkListType: learn
         links:
-          - text: Quantum computing concepts
-            url: concepts/index.md
-          - text: Glossary
-            url: Glossary.md
+          - text: Learn with the Quantum Katas
+            url: intro-to-katas.md
+          - text: Browse Q# code samples
+            url: https://docs.microsoft.com/samples/browse/?languages=qsharp
```

Comment:
Maybe we can add here also a link to "How to learn quantum computing?" since it's our best compilation of learning resources

Reply:
I'm leaning away from that, as I think it would definitely be over-saturating to have the same link in two nearby places on the same page. Also, this card is more for resources directly relating to learning about/with Q#, so for those already diving into the coding